src/Components/Calculator.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import Button from './Button'
+import buttons from '../assets/calculatorKeys'
+
+
+
+// console.log(buttonKeys)
+// buttonKeys.map(buton=>(
+// 	{/* <Button
+// 		key={buton.value}
+// 		number={buton.number}
+// 		type={buton.type}
+// 	/> */}
+// 	console.log(buton);
+// ))
+
+// buttons.map((button)=>{
+// console.log(button)
+// })
```

Comment:
- [ ] Kindly remove all unnecessary commented-out codes from your file. Doing so makes your code look clean and readable. 👍

Reply:
This change has been done well